kubernetes/overlays/agreement-readmodel-writer/dev-refactor/configmap.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{SERVICE_NAME}}
+  namespace: {{NAMESPACE}}
+data:
+  KAFKA_CLIENT_ID: "{{NAMESPACE}}-pagopa-interop-agreement"
```

Comment:
this is how the config is set for agreement and attribute, but shouldn't it be `{{NAMESPACE}}-pagopa-interop-agreement-readmodel-writer`?

Reply:
I am not sure, but isn't your suggestion solved by the groupId?
The KAFKA_CLIENT_ID should be more generic and close to the domain name.

Reply:
given that client id has no practical utility, the documentations says

```
A client ID in Kafka is a label you define that names a particular consumer or producer. You can give your client a friendly name so that debugging is easier.
...
Client IDs should not be confused with group IDs. A group ID will affect the way records are consumed, but a client ID is just a label.
```

in theory each consumer/producer should have its own client id